docs/tutorials/excel-tutorial.md

```diff
@@ -51,47 +51,58 @@ In this step of the tutorial, you'll programmatically test that your add-in supp
 
 1. Open the project in your code editor.
```

Comment:
There is confusion at this point. The generator finishes with 3 next steps that tell the user to start the server and open VS Code. But the tutorial tells the user to open the project in "your code editor" and says nothing about starting the server. We need to either (1) tell the reader in the tutorial to ignore the instructions at the end of the generator, or (2) change the tutorial so that it assumes that the reader has carried out the next steps that the generator provides, or (3) change the generator so that it no longer has these next steps. **I prefer the third option.** This option maximizes the flexibility of contexts in which we can use the generator. 
![image](https://user-images.githubusercontent.com/9088056/64999561-1d076600-d89d-11e9-92cc-4c755a4dfbca.png)

Reply:
Valid point re the confusion at this point. Seems like this guidance that the generator provides after the project's been created needs to remain as-is, so that folks who run the generator independently (i.e., while *not* following along with any of our documentation) will know what to do at that point. I'll add info to the tutorials (and to other places in the docs that contain Yo Office instructions) suggesting that the "next steps" that the generator provides can be ignored.